blog.html

```diff
@@ -45,25 +45,25 @@ <h2>Our blog</h2>
         <section>
           <h2>Categories</h2>
           <ul class="blog-navigation__list">
```

Comment:
looks like should be only "blog-navigation" here, because inside there are some elements